src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs

```diff
@@ -107,5 +109,53 @@ public void TestNoCyclesInRefactoringProviders()
             actualOrder = ExtensionOrderer.Order(vbProviders).ToArray();
             Assert.True(actualOrder.Length > 0);
         }
+
+        [Fact, WorkItem(1599579, "https://devdiv.visualstudio.com/DevDiv/_queries/edit/1599579")]
+        public void TestCodeFixServiceOrderIsCorrect()
+        {
+            // This test will fail if a cycle is detected in the ordering of our code fix providers.
+            // If this test fails, you can break the cycle by inspecting and fixing up the contents of
+            // any [ExtensionOrder()] attributes present on our code fix providers.
+            var providers = ExportProvider.GetExports<CodeFixProvider, CodeChangeProviderMetadata>();
+            var providersPerLanguage = providers.ToPerLanguageMapWithMultipleLanguages();
+
+            TestCodeFixServiceOrderIsCorrectForLanguage(LanguageNames.CSharp, providersPerLanguage);
+            TestCodeFixServiceOrderIsCorrectForLanguage(LanguageNames.VisualBasic, providersPerLanguage);
```

Comment:
❓is `InlineData` better here so it's a single test failure per language?

Reply:
sure :)

Reply:
yup.  done.